modules/generate-network-policies-ng20.adoc

```diff
@@ -10,17 +10,12 @@
 
 You can generate network policies from the network graph.
 
-The generated policies apply to the deployments shown in the network graph and they allow all network traffic observed during the selected time.
+The generated policies apply to all deployments that exist in the currently selected cluster and they allow all network traffic observed during the baseline discovery period.
```

Comment:
```suggestion
The generated policies apply to all of the deployments that exist in the currently selected cluster and they allow all network traffic observed during the baseline discovery period.
```

I'm wondering if we should  make this two sentences though (or not. Up to you)

```suggestion
The generated policies apply to all of the deployments that exist in the currently selected cluster. They also allow all network traffic observed during the baseline discovery period.
```

or 

```suggestion
The generated policies apply to all of the deployments that exist in the currently selected cluster. Generated policies also allow all network traffic observed during the baseline discovery period.
```